content/why/_index.md

```diff
@@ -96,10 +96,85 @@ There should be a self reinforcing energy to all of this: we want tools that wor
 Strategy
 --------
 
+### familiar tools
+
+Specifically, the primary target and scope can be summarized as “must support linux-like systems” and “must pass state primarily by familiar ‘file’-like abstractions”.
+(In other words -- “it should taste like containers” -- this is a proven path to successful adoption that lets others use what they know and pivot towards our tooling incrementally.  Incremental is good.)
+This scope may extend to more systems in the future, but will start here.
+
+### anything not everything
+
+The Timeless Stack tools need to support *anything* -- anything you want to put in them, with no special consideration required -- rather than supporting "everything", but needing plugins to do it.  This stands in contrast to most other build tools.
+
 ### breaking down the problem
 
+There are five major sets of problems that need to be addressed:
+
+1. Framing and hermetic execution and fileset-slinging core
```

Comment:
This sentence doesn't make a lot of sense and needs more clarity.
What is `framing`? I'm going to make an assumption that it means the composability of file sets
```suggestion
1. Fileset transport: Content addressable filesets can be discovered, fetched, and verified.
2. Framing: File sets can be composed together into a working tree.
3. Hermetic execution: Transformations to the working tree are executed with minimal side-effects. The primary side-effect should be the specified file set export.
```

Reply:
Yeah something like this change is getting closer... the "slinging" part in particular is a hangover from an earlier flippant draft >.<

The idea I had in mind with "Framing" is to start building up a contrast between "build tools" and "framing tools" -- maybe "framing" aka "workspace" -- generally trying to get at the idea that we want to make general purpose contains-anything systems rather than getting into the nitty-gritty of every individual thing that gets built.  Urban planning vs pouring concrete, maybe; some idea like that.  So composing filesets is part of it, but I'd like to make it more general as well.

(Maybe this is a lot of work to try to make one word do, so early in this intro-audience document...)

I'll cook on this in the back of my mind for a while longer and then probably take most of this diff you propose, but if you get additional ideas, also jot em :)